app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -122,6 +122,7 @@ function findIncorrectArgsLength(content) {
         let argumentMatch
         while ((argumentMatch = /\(/g.exec(argumentsString)) != null && safeIndex < 100) {
           const argumentClosing = getClosingBracket(argumentsString, "(", ")", argumentMatch.index - 1)
+          if (argumentClosing === -1) { break }
```

Comment:
I feel like Mitsiee would ask for this, so let me ask for it first 😁

```suggestion
          if (argumentClosing === -1) break
```

Reply:
are there coding standards for this that should be followed or is it just personal preference? 
i assume its because its single line that the clamps shouldnt be there? 
if nothing else i suppose its better to remove them just for the sake of keeping it consistent with the rest of the code

Reply:
I guess its Mitsiee's personal style. That is to say, its the code style of the project since he's the owner 😄.

And indeed this is because the if-statement only contains one statement inside. I thought ESLint would catch this, but I guess not.

Reply:
Indeed, it's just a preference things. I will see if I can set up ESLint or prettier a little better, ideally they would catch all of these things